interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -517,6 +530,11 @@ protected ManagedChannelBuilder<?> createChannelBuilder() {
           channelCredentials = InsecureChannelCredentials.create();
         }
       }
+      if (TestCases.ORCA.equals(testCase)) {
+        customBackendMetricsLoadBalancerProvider = new CustomBackendMetricsLoadBalancerProvider();
```

Comment:
With some of the other changes discussed, we could make this a custom LB that is always registered. That could also help solve the protobuf+android problem, as we could just mark the provider as unavailable or avoid registering it when running on android.